lib/active_model/serializer/adapter.rb

```diff
@@ -0,0 +1,59 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      class << self
+        def create(resource, options = {})
+          warn "Calling deprecated ActiveModel::Serializer::Adater in #{caller[0..2].join(', ')}. Please use ActiveModelSerializers::Adapter"
+          ActiveModelSerializers::Adapter.create(resource, options)
+        end
+
+        # @see ActiveModel::Serializer::Adapter.lookup
+        def adapter_class(adapter)
+          warn "Calling deprecated ActiveModel::Serializer::Adater in #{caller[0..2].join(', ')}. Please use ActiveModelSerializers::Adapter"
+          ActiveModelSerializers::Adapter.adapter_class(adapter)
+        end
+
+        # @return Hash<adapter_name, adapter_class>
+        def adapter_map
+          warn "Calling deprecated ActiveModel::Serializer::Adater in #{caller[0..2].join(', ')}. Please use ActiveModelSerializers::Adapter"
+          ActiveModelSerializers::Adapter.adapter_map
+        end
+
+        # @return [Array<Symbol>] list of adapter names
+        def adapters
+          warn "Calling deprecated ActiveModel::Serializer::Adater in #{caller[0..2].join(', ')}. Please use ActiveModelSerializers::Adapter"
+          ActiveModelSerializers::Adapter.adapters
+        end
+
+        # Adds an adapter 'klass' with 'name' to the 'adapter_map'
+        # Names are stringified and underscored
+        # @param name [Symbol, String, Class] name of the registered adapter
+        # @param klass [Class] adapter class itself, optional if name is the class
+        # @example
+        #     AMS::Adapter.register(:my_adapter, MyAdapter)
+        # @note The registered name strips out 'ActiveModel::Serializer::Adapter::'
+        #   so that registering 'ActiveModel::Serializer::Adapter::Json' and
+        #   'Json' will both register as 'json'.
```

Comment:
no need for all the comments here. Can just be `{ActiveModelSerializers::Adapter.register}` I think